docs/getting-started/kubernetes-operator.md

```diff
@@ -34,32 +36,31 @@ By default, the operator will be installed in the `dragonfly-operator-system` na
 
 ## Usage
 
-### Creating a Dragonfly instance
+### Create a Dragonfly cluster
```

Comment:
This is actually not a cluster, but an instance with replicas.
A cluster is a term used by Redis (and soon Dragonfly) to split the keyspace between different masters, which this section does not cover.
However, I do see why "instance" is not the best term here. How about "node"? or "instance with replicas"?

Reply:
BTW the same is true for the below changes as well